### DISCUSSION TITLE
System.Net tracing in Powershell Core on Linux

### DISCUSSION THREAD

**Discussion by p0macs** [PowerShell/PowerShell#16744] [Q&A]
I have some issues with SSL in a Powershell script, running in a docker container.
There are some docs about how to enable network tracing for .Net - like this:
https://techcommunity.microsoft.com/t5/iis-support-blog/troubleshooting-tls-ssl-communication-problems-when-making-http/ba-p/481441

I also saw similar solution using a powershell.exe.config file for Powershell desktop on Windows.

I'm curious if similar tracing is possible on Powershell Core for Linux?

Can anybody help me?

Thanks!

**Comment by iSazonov** [comment]
I hope follow helps you https://github.com/dotnet/diagnostics/blob/main/documentation/dotnet-trace-instructions.md

**Reply by p0macs** [reply]
Thank you for the hint, but unfortunately this does not works for me.
All what I have is a docker image with Powershell Core, from here: https://hub.docker.com/_/microsoft-powershell?tab=description
There is no dotnet framework installed in the image, and I cannot install additional software in the image. I could copy the dotnet-trace executable to the container, but it will not work without a dotnet framework:

```
It was not possible to find any compatible framework version
The framework 'Microsoft.NETCore.App', version '3.1.0' (x64) was not found.
  - No frameworks were found.
```

**Reply by iSazonov** [reply]
I think you get answer in .Net repository more fast.
